src/main.rs

```diff
@@ -9,59 +9,63 @@ use std::env;
 use native_dialog::MessageDialog;
 use native_dialog::MessageType;
 use webbrowser;
+use std::path::{PathBuf, Path};
+use std::ffi::OsString;
 
 fn main() {
     if let Ok(dir) = get_minecraft_installation_dir() {
-        println!("Minecraft install dir {}", dir);
+        println!("Minecraft install dir {}", dir.to_str().unwrap());
```

Comment:
Rather than panicking on invalid UTF-8 here, should probably use `dir.display()`

Reply:
Oooh, didn't know about that existing! Thanks, will change it.